static/css/impala/devhub-api.less

```diff
@@ -12,8 +12,6 @@
     label {
       font-weight: bold;
       display: block;
-      width: 10%;
```

Comment:
I find problem from your solution, because this input if remove width we not see full api-key
![2017-02-10 13-13-51](https://cloud.githubusercontent.com/assets/6254062/22823396/d37d9e50-ef9c-11e6-8113-8659f41e0b5c.png)
 If i set width: 100% show up over width:
![2017-02-10 13-14-32](https://cloud.githubusercontent.com/assets/6254062/22823447/1d7aadc2-ef9d-11e6-9e81-953ee819029f.png)

Resolution: solution where 78% width is good way :)
![2017-02-09 20-14-34](https://cloud.githubusercontent.com/assets/6254062/22823476/45a9b0b8-ef9d-11e6-9edb-61bb7d10e22a.png)